12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -4,6 +4,7 @@ taxonomy:
     category:
         - docs
 ---
+Edited and fixed by BestNoob
```

Comment:
Not really necessary.

Reply:
doesnt know ^^ just want to see my name there, a piece of fame, a piece of yes, thank you BestNoob you did good changes to this, just becasue of your infos i can create my mod xD